### DISCUSSION TITLE
React 18 support

### DISCUSSION THREAD

**Discussion by AhmedBenGayess** [segmentio/evergreen#1452] [Q&A] [answered]
Hello and thank you for your work 😄 
Is support for React 18 coming soon or should I stick with react 17?

**Comment by brandongregoryscott** [comment, marked as answer]
Moving this over to #1453

FWIW, it should _work_ with React v18 as-is (I'm using it in a React v18 app myself) but it will produce the console warning about using `ReactDOM.render`.

Update 6/21/22: [v6.10.0](https://github.com/segmentio/evergreen/releases/tag/v6.10.0) has been released which includes React 18+ in the peer dependency range, which should alleviate the need to install with `--force` or `--legacy-peer-deps` when using a newer npm version.
